### PR TITLE
Update researcher-server-access-footer.md

### PR DESCRIPTION
**Pull request by LiamHart-hub**
Update on the Safe Researcher Training to provide more clarity to those that apply.

### CODE REVIEW

File: snippets/researcher-server-access-footer.md
Comment (iaindillingham) [inline snippets/researcher-server-access-footer.md:1]: One suggestion about links, @LiamHart-hub. You need not display the URL in the text. You can instead use a more meaningful word or phrase to describe the link. Compare the following passages:

Passage one:

> If you do not have such training, you must register with the ONS' Research Accreditation Service (RAS); use this link to register on the RAS platform: https://researchaccreditationservice.ons.gov.uk/ons/ONS_homepage.ofml.

Passage two:

> If you do not have such training, you must register with the ONS' [Research Accreditation Service (RAS)][1].

There are several advantages to passage two when compared to passage one. However, it's important to emphasize that the word or phrase is meaningful. ("Click here" links, which you may see elsewhere on the Internet, are not meaningful.)

[1]: https://researchaccreditationservice.ons.gov.uk/ons/ONS_homepage.ofml

**Review by iaindillingham — APPROVED** [review]
I've made a comment about links, which I'd like you to consider.

File: snippets/researcher-server-access-footer.md
Comment (LiamHart-hub) [reply snippets/researcher-server-access-footer.md:1]: @iaindillingham, I am happy to take on your suggestions and add the 'Click here' wording to improve the readability of the text

Comment (iaindillingham) [reply snippets/researcher-server-access-footer.md:1]: Thanks, @LiamHart-hub. Just to be clear: please **don't** add "Click here" links. Please, instead, use a meaningful word or phrase.

Comment (LiamHart-hub) [reply snippets/researcher-server-access-footer.md:1]: Okay. Noted. I will make sure this approach is taken in the future. Thank you for your support on this, Iain.

**Pull request merged**